packages/backend/server/src/plugins/worker/controller.ts

```diff
@@ -77,6 +91,10 @@ export class WorkerController {
       const contentType = response.headers.get('Content-Type');
       const contentDisposition = response.headers.get('Content-Disposition');
       if (contentType?.startsWith('image/')) {
+        await this.cache.set(
+          targetURL.toString(),
+          await response.arrayBuffer()
```

Comment:
It is necessary to add an expiration time to prevent the cache from growing larger and larger